src/Lykke.Service.HFT/Modules/ServiceModule.cs

```diff
@@ -65,13 +65,13 @@ protected override void Load(ContainerBuilder builder)
 
             RegisterOrderBooks(builder);
 
-            RegisterAssets(builder);
-
             RegisterOrderStates(builder);
 
             BindRabbitMq(builder, currentSettings.HighFrequencyTradingService);
 
-            RegisterPeriodicalHandlers(builder);
+#if !DEBUG
+            RegisterPeriodicalHandlers(builder); 
```

Comment:
is this `if DEBUG` needed?

Reply:
This is a long-term process that can influence other instances. I wouldn't like this code to run when I am debugging.